pom.xml

```diff
@@ -267,6 +267,44 @@
         <activeByDefault>true</activeByDefault>
       </activation>
     </profile>
+    <profile>
+        <id>apache-release</id>
+        <build>
+            <plugins>
+                <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-assembly-plugin</artifactId>
+          <dependencies>
+              <dependency>
+                  <groupId>org.apache.apache.resources</groupId>
+                  <artifactId>apache-source-release-assembly-descriptor</artifactId>
```

Comment:
This is the same as the official Apache one:
https://github.com/apache/maven-apache-parent/blob/master/pom.xml#L381